src/common/components/Radio/Radio.tsx

```diff
@@ -1,20 +1,55 @@
-import * as React from 'react'
+import React, { useEffect, useState } from 'react'
+import cn from 'classnames'
 
-import * as Styles from './Radio.styles'
-
-interface RadioProps {
+export type TRadioProps = {
   isChecked: boolean
-  clickHandler: (event: React.MouseEvent<HTMLLabelElement, MouseEvent>) => void
+  clickHandler: (param: boolean) => void
+  className?: string
+  smallCircleClass?: string
   children: React.ReactNode
 }
 
-function Radio({ children, isChecked, clickHandler }: RadioProps): JSX.Element {
+const Radio = ({
+  children,
+  isChecked,
+  clickHandler,
```

Comment:
Better to have more conventional names, like in standard checkbox or radio inputs: `checked` and `onChange`.
Please update if that sounds good to you

Reply:
fixed